test/integration/targets/vyos_l3_interfaces/tests/cli/overridden.yaml

```diff
@@ -0,0 +1,52 @@
+---
+- debug:
+    msg: "START vyos_l3_interfaces merged integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate.yaml
+
+- name: Overrides all device configuration with provided configuration
+  vyos_l3_interfaces: &overridden
+    config:
+      - name: eth0
```

Comment:
I don't believe we want to use this interface for testing, as it is mgmt

Reply:
@pabelanger The reason for mentioning this interface in the task is to make sure that overridden doesn't cleanup eth0's IP configurations. This task, in fact, doesn't execute any command to modify eth0.